router/router_test.go

```diff
@@ -275,3 +282,22 @@ func TestValidateDefaultAliases(t *testing.T) {
 		}
 	}
 }
+
+func TestBidderParamsCompactedOutput(t *testing.T) {
+	expectedPrefix := `{"33across":{"$schema":"http://json-schema.org/draft-04/schema#","title":"33Across Adapter Params","description":"A schema which validates params accepted by the 33Across adapter","type":"object","properties":{"productId":{"type":"string","description":"Product type"},"siteId":{"type":"string","description":"Site Id"},"zoneId":{"type":"string","description":"Zone Id"}},"required":["productId","siteId"]}`
```

Comment:
Does this rely on the real 33across static file? Will this test fail if that bidder makes an update?

Reply:
Yes, I forgot to swap the real file for a proper JSON test file. Modified.